services/galaxytoolshed/galaxy-toolshed-downloads.service.js

```diff
@@ -0,0 +1,55 @@
+import Joi from 'joi'
+import { renderDownloadsBadge } from '../downloads.js'
+import BaseGalaxyToolshedService from './galaxy-toolshed-base.js'
+
+const repositoryRevisionInstallInfoSchema = Joi.array().items(
+  Joi.object({
+    times_downloaded: Joi.number().required(),
+  }).required(),
+  Joi.object({}),
+  Joi.object({})
+)
+
+export default class GalaxyToolshedDownloads extends BaseGalaxyToolshedService {
+  static category = 'downloads'
+  static route = {
+    base: 'galaxy-toolshed/downloads',
+    pattern: ':repository/:owner',
+  }
+
+  static examples = [
+    {
+      title: 'Galaxy Toolshed - Downloads',
+      namedParams: {
+        repository: 'sra_tools',
+        owner: 'iuc',
+      },
+      staticPreview: this.render({ downloads: 10000 }),
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'downloads',
+  }
+
+  static render({ downloads }) {
+    return renderDownloadsBadge({ downloads })
+  }
+
```

Comment:
We don't need a dedicated `render` function if we're just passing through directly to a render helper like this 

```suggestion
```